src/components/Home/Home.tsx

```diff
@@ -46,8 +46,8 @@ const Home: FC<Props> = ({
         <>
             <ContactInfo contact={contact} companies={companies} owner={contactOwner} />
             <Deals deals={deals} owners={dealOwners} accountInfo={accountInfo} dealPipelines={dealPipelines} />
-            {notes.length > 0 && <Notes notes={notes} owners={noteOwners} />}
-            {(callActivities.length > 0 || emailActivities.length > 0) && <Activities calls={callActivities} emails={emailActivities} />}
+            <Notes notes={notes} owners={noteOwners} />
```

Comment:
Here discussion in slack about activities and visibility sections https://deskpro.slack.com/archives/C4AQRPD8T/p1665999505211669